backend/flask_routes/app.py

```diff
@@ -680,9 +681,103 @@ def filter_existing_goals(create_goal_request):
     return result
 
 
+"""
+helper to send users notifications that update on the state of the goals they're currently tracking
+following a sync to grab users most recent transaction data from plaid
+"""
+def send_goal_update_msg(userId):
+    user = User.query.get(userId)
+    message_body = 'Here\'s your breakdown on goal progress since last week: \n' + format_updates(userId)
```

Comment:
allowing users to set notification preferences (i.e. how often they want to be notified) would be a good way to extend this

Reply:
yeah definitely, I think sending in line with when transaction data is updated is the natural place to start and it works well but this could be an improvement I get in tomorrow